src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -40,6 +40,20 @@ export const GithubCreateBranchPost = async (req: Request, res: Response): Promi
 		res.sendStatus(200);
 	} catch (err) {
 		req.log.error({ err }, "Error creating branch");
-		res.sendStatus(500);
+		res.status(500).json(verifyError(err));
```

Comment:
If we have the correct code can we want it pass it in, in place of the 500?

Reply:
Updated